scripts/transform_rnaseq.py

```diff
@@ -23,36 +23,30 @@
 import argparse
 from subprocess import call
 import pandas as pd
-from process_rnaseq import zero_one_normalize
+from process_rnaseq import normalize_data
 
-####################################
 # Load Command Arguments
-####################################
-parser = argparse.ArgumentParser()  # Load command line options
-parser.add_argument("-m", "--microarray-fh", dest="microarray_fh",
-                    help="file handle for microarray data")
-parser.add_argument("-r", "--rnaseq-fh", dest="rnaseq_fh",
-                    help="file handle for RNAseq data")
-parser.add_argument("-o", "--out-fh", dest="out_fh",
-                    help="file handle for normalized output file")
+parser = argparse.ArgumentParser()
+parser.add_argument("-m", "--microarray_file", dest="microarray_file",
+                    help="file name for microarray data")
+parser.add_argument("-r", "--rnaseq_file", dest="rnaseq_file",
+                    help="file name for RNAseq data")
+parser.add_argument("-o", "--out_file", dest="out_file",
+                    help="file name for normalized output file")
 parser.add_argument("-n", "--normalization", dest="normalization",
                     help="the normalization method")
 args = parser.parse_args()
 
-####################################
 # Load Constants
-####################################
-MICRO_FH = args.microarray_fh
-RNASEQ_FH = args.rnaseq_fh
-OUT_FH = args.out_fh
-NORMALIZATION = args.normalization
+micro_file = args.microarray_file
+rnaseq_file = args.rnaseq_file
+out_file = args.out_file
+normalization = args.normalization
 
-####################################
 # Analysis
-####################################
-command = 'R --no-save --args ' + MICRO_FH + ' ' + RNASEQ_FH + \
-          ' ' + OUT_FH + ' < ' + 'scripts/util/transform.r'
+command = 'R --no-save --args {} {} {} < \
+          scripts/util/transform.R'.format(micro_file, rnaseq_file, out_file)
 call(command, shell=True)
 
-X = pd.read_csv(OUT_FH, delimiter='\t', index_col=0)
-zero_one_normalize(X, OUT_FH, method=NORMALIZATION)
+X = pd.read_csv(out_file, delimiter='\t', index_col=0)
```

Comment:
I definitely agree - thanks for catching! I've updated the variable names in the most recent push